cmd/observer/node_builder/observer_service_builder.go

```diff
@@ -217,17 +218,17 @@ func (builder *UnstakedAccessNodeBuilder) initLibP2PFactory(networkKey crypto.Pr
 	}
 }
 
-// initUnstakedLocal initializes the unstaked node ID, network key and network address
+// initUnstakedLocal initializes the observer node ID, network key and network address
 // Currently, it reads a node-info.priv.json like any other node.
 // TODO: read the node ID from the special bootstrap files
-func (builder *UnstakedAccessNodeBuilder) initUnstakedLocal() func(node *cmd.NodeConfig) error {
+func (builder *ObserverServiceBuilder) initUnstakedLocal() func(node *cmd.NodeConfig) error {
 	return func(node *cmd.NodeConfig) error {
-		// for an unstaked node, set the identity here explicitly since it will not be found in the protocol state
+		// for an observer node, set the identity here explicitly since it will not be found in the protocol state
```

Comment:
Sure yes. I did this one with Shawn too early, so that that code does not apply cleanly anymore.
I decided rather to do a quick back and forth, to ensure the style changes reflect your preference.
Fixed in https://github.com/GetElastech/flow-go/commit/954e51f0d934fba6b516b0668bf052903e521d8d